### PR TITLE
pythonPackages.python-logstash: init at 0.4.6

### PR DESCRIPTION
**Pull request by peterromfeldhk**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/python-modules/python-logstash/default.nix [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "13763yx0k655y0c8gxv7jj6cqp45zypx2fmnc56jnn9zz1fkx50h";
+  };
+
+  nativeBuildInputs = [ pika ];
+
+  meta = with lib; {
+    description = "Python logging handler for Logstash. http://logstash.net/";
```
Comment (worldofpeace) [inline pkgs/development/python-modules/python-logstash/default.nix]: Putting links is the description isn't helpful.
That would be better to have in `homepage`.

Also having `maintainers` set is good thing to have too.

Comment (worldofpeace) [reply pkgs/development/python-modules/python-logstash/default.nix]: Maybe also add a `repositories.git` with `https://github.com/vklochan/python-logstash`.

File: pkgs/development/python-modules/python-logstash/default.nix [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  pname = "python-logstash";
+  version = "0.4.6";
+
+  disabled = isPyPy;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "13763yx0k655y0c8gxv7jj6cqp45zypx2fmnc56jnn9zz1fkx50h";
+  };
+
```
Comment (worldofpeace) [inline pkgs/development/python-modules/python-logstash/default.nix]: I don't see any tests in the repo so we should disable the `checkPhase` like
```
doCheck = false;
```

**Review by worldofpeace — DISMISSED** [review]

File: pkgs/development/python-modules/python-logstash/default.nix [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  doCheck = false;
+
+  meta = with lib; {
+    description = "Python logging handler for Logstash";
+    homepage = "https://github.com/vklochan/python-logstash";
```
Comment (dotlambda) [inline pkgs/development/python-modules/python-logstash/default.nix]:
```suggestion
    homepage = https://github.com/vklochan/python-logstash;
```

File: pkgs/development/python-modules/python-logstash/default.nix [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+  version = "0.4.6";
+
+  disabled = isPyPy;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "13763yx0k655y0c8gxv7jj6cqp45zypx2fmnc56jnn9zz1fkx50h";
+  };
+
+  doCheck = false;
```
Comment (dotlambda) [inline pkgs/development/python-modules/python-logstash/default.nix]: Add a comment with the reason.

File: pkgs/development/python-modules/python-logstash/default.nix [hunk 5]
```diff
@@ -0,0 +1,22 @@
+{ stdenv, lib, buildPythonPackage, isPyPy, fetchPypi }:
+
+buildPythonPackage rec {
+  pname = "python-logstash";
+  version = "0.4.6";
+
+  disabled = isPyPy;
```
Comment (dotlambda) [inline pkgs/development/python-modules/python-logstash/default.nix]: Do you set this for a reason?

**Review by dotlambda — DISMISSED** [review]

File: pkgs/development/python-modules/python-logstash/default.nix (hunk 5 above)
Comment (peterromfeldhk) [reply pkgs/development/python-modules/python-logstash/default.nix]: it was in the example.. and i am not sure how to test if pypy is working

Comment (dotlambda) [reply pkgs/development/python-modules/python-logstash/default.nix]: No need to test. Just don't set it without a reason.

File: pkgs/development/python-modules/python-logstash/default.nix [hunk 6]
```diff
@@ -0,0 +1,21 @@
+{ stdenv, lib, buildPythonPackage, isPyPy, fetchPypi }:
```
Comment (worldofpeace) [inline pkgs/development/python-modules/python-logstash/default.nix]:
```suggestion
{ stdenv, lib, buildPythonPackage, fetchPypi }:
```

**Comment by mmahut** [comment]
Are there any updates on this pull request, please?

**Comment by marsam** [comment]
Sorry for the delay. Thank you @peterromfeldhk

**Pull request merged**